### ISSUE TITLE
Giants and Dwarves TD - Can't place towers

### ISSUE THREAD

**Issue by desertking** [ruffle-rs/ruffle#11640]
### Describe the bug

Dunno why but in the game I'm not able to place the tower. Doesn't recognize the click on the spot it seems.
https://www.newgrounds.com/portal/view/621555?emulate=flash

### Expected behavior

Able to place towers

### Affected platform

Online demo

### Operating system

All

### Browser

All

**Comment by n0samu** [comment]
The game also breaks trying to load an existing save slot:
```
ERROR run_frame:run_all_phases_avm2: ruffle_core::avm2::events: Error dispatching event EventObject(EventObject { type: "enterFrame", class: flash.events::Event, ptr: 0x22fb2a04988 }) to handler FunctionObject(FunctionObject { ptr: 0x22fb2a850f8 }) : TypeError: Error #1010: A term is undefined and has no properties. (accessing field: length)
        at gGame$/LoadGameData()
        at mcSaveSlotSelectCls$/InitSlot()
        at mcSaveSlotSelectCls/UpdateSlots()
        at mcSaveSlotSelectCls()
        at IGame/OpenSaveSlotSelect()
        at mcMainMenuCls/StartCampaign()
        at PageTurn/OnEnterFrame()
```

**Comment by n0samu** [comment]
The saving/loading issue is due to unimplemented VectorObject serialization, like #9688
`WARN ruffle_core::avm2::amf: Serialization is not implemented for class other than Object: VectorObject(VectorObject { ptr: 0x1f69ffd5dc0 })`

**Comment by n0samu** [comment]
And the tower-placing issue happens because `getObjectsUnderPoint` is unimplemented. This will be fixed by #11970.
`WARN run_frame:run_all_phases_avm2: ruffle_core::stub: Encountered stub: AVM2 flash.display.DisplayObjectContainer.getObjectsUnderPoint()`

**Comment by serkonda7** [comment]
This issue can be closed.
